Source/PrivateHeaders/Ably/ARTResumeRequestResponse.h

```diff
@@ -0,0 +1,72 @@
+@import Foundation;
+
+@class ARTProtocolMessage;
+@class ARTErrorInfo;
+@protocol ARTErrorChecker;
+
+/**
+ The type of the Realtime system’s response to a resume request, as described by RTN15c.
+ */
+typedef NS_ENUM(NSUInteger, ARTResumeRequestResponseType) {
+    /**
+     RTN15c6: "This indicates that the resume attempt was valid. The client library should move all channels that were in the @ATTACHING@, @ATTACHED@, or @SUSPENDED@ states to the @ATTACHING@ state, and initiate an @RTL4c@ attach sequence for each. The connection should also process any messages queued per @RTL6c2@ (there is no need to wait for the attaches to finish before processing queued messages)."
```

Comment:
```suggestion
     RTN15c6: "This indicates that the resume attempt was valid. The client library should move all channels that were in the `ARTRealtimeChannelAttaching`, `ARTRealtimeChannelAttached`, or `ARTRealtimeChannelSuspended` states to the `ARTRealtimeChannelAttaching` state, and initiate an RTL4c attach sequence for each. The connection should also process any messages queued per RTL6c2 (there is no need to wait for the attaches to finish before processing queued messages)."
```

Reply:
I've changed the Textile `@` to a more readable <code>`</code> instead, but apart from that I'd like to just keep it the same as the feature spec — the quotation marks indicate that it's just a copy and paste.